src/App.tsx

```diff
@@ -1,12 +1,12 @@
-import 'bootstrap/dist/css/bootstrap.min.css';
 import React, { FC } from 'react';
-import { HomePage } from './components/HomePage/HomePage';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import ErrorBoundary from './components/shared/ErrorBoundary';
+import { HomePage } from './components/HomePage/HomePage';
+import { MovieDetailsPage } from './components/MovieDetailsPage/MovieDetailsPage';
 
-export const App: FC = () => {
-  return (
-    <ErrorBoundary>
-      <HomePage />
-    </ErrorBoundary>
-  );
-};
+export const App: FC = () => (
+  <ErrorBoundary>
+    {/* <HomePage /> */}
```

Comment:
Is it temporary?

Reply:
Yes, it is. It will be updated during the next task with routing.